.github/workflows/main.yaml

```diff
@@ -51,10 +63,11 @@ jobs:
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@171dd555b9ab6b18fa02519fdfacbb8bf671e1b4  # locking to a commit version for now
         with:
-          branch: automation/add-new-dotnet-release
-          title: New DotNetCore Sdks
+          branch: automation/add-new-dotnet-release-${{ env.releaseDate }}
+          title: New DotNetCore Sdks ${{ env.releaseDate }}
 
       - name: Run Azure Pipeline 
         run: |
           echo  "${{ secrets.AZURE_DEVOPS_TOKEN }}" | az devops login --organization https://devdiv.visualstudio.com/
-          az pipelines run --organization https://devdiv.visualstudio.com/ --project DevDiv --name Oryx-PlatformBinary-DotNetCore --branch automation/add-new-dotnet-release
\ No newline at end of file
+          az pipelines run --organization https://devdiv.visualstudio.com/ --project DevDiv --name Oryx-PlatformBinary-DotNetCore --branch automation/add-new-dotnet-release-${{ env.releaseDate }}
+          az pipelines run --organization https://devdiv.visualstudio.com/ --project DevDiv --name Oryx-Validation --branch automation/add-new-dotnet-release-${{ env.releaseDate }}
```

Comment:
Appending the date allows us to prevent new SDKs to get overwritten by no other changes. For example, if V1 is released today and V2 is released tomorrow, then V1 will have it's own branch and V2 will have it's own branch. Preventing V2 from overwriting V1 changes.